homeassistant/components/cloud/backup.py

```diff
@@ -208,6 +210,7 @@ async def async_list_backups(self, **kwargs: Any) -> list[AgentBackup]:
         """List backups."""
         try:
             backups = await async_files_list(self._cloud, storage_type=_STORAGE_BACKUP)
+            _LOGGER.debug("Cloud backups: %s", backups)
```

Comment:
This can/will become large and should be guarded so it's not sent to the logger unless debug is enabled for this logger.

Reply:
It is already? As in, this is lazy logged?

Reply:
So we can remove all of these (`_LOGGER.isEnabledFor(logging.DEBUG)`) then [example](https://github.com/home-assistant/core/blob/94ad6ae814fe2bea4cfc726d62efe92a13fcab61/homeassistant/components/tag/__init__.py#L221).